msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2024-05-07 00:16+0000\n"
"PO-Revision-Date: 2024-05-21 18:25\n"
"Last-Translator: \n"
"Language-Team: Norwegian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: no\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: no_NO\n"

#: pylav/extension/bundled_node/manager.py:239
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture."
msgstr "Du prøver å kjøre den administrerte Lavalink-noden på en maskinarkitektur som ikke støttes."

#: pylav/extension/bundled_node/manager.py:466
msgid "The output of `{command_name_variable_do_not_translate}` was unexpected\n"
"{command_output_variable_do_not_translate}"
msgstr "Utgangen av '{command_name_variable_do_not_translate}' var uventet\n"
"{command_output_variable_do_not_translate}"

#: pylav/extension/bundled_node/manager.py:489
msgid "The port {port_variable_do_not_translate} is already in use. Managed Lavalink startup has been aborted."
msgstr "Havnen {port_variable_do_not_translate} er allerede i bruk. Administrert Lavalink-oppstart er avbrutt."

#: pylav/extension/bundled_node/manager.py:494
msgid "Lavalink failed to start: {error_variable_do_not_translate}"
msgstr "Lavalink kunne ikke starte: {error_variable_do_not_translate}"

